faiss/gpu/impl/CuvsIVFPQ.cu

```diff
@@ -229,6 +229,7 @@ void CuvsIVFPQ::searchPreassigned(
         Tensor<idx_t, 2, true>& outIndices,
         bool storePairs) {
     // TODO: Fill this in!
```

Comment:
Can you create an issue and reference the issue number here just so we don't lose sight of this?

Reply:
Done!